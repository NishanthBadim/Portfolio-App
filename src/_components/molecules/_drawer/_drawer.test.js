import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Drawer from "./_drawer";
import { BrowserRouter } from "react-router-dom";

describe("Drawer testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<BrowserRouter><Drawer /></BrowserRouter>);
    expect(input.toJSON()).toMatchSnapshot();
  })
})