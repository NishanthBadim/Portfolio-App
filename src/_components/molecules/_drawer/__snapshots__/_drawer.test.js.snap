// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer testing Matches the snapshot 1`] = `
<div
  className="mdl-layout__drawer"
>
  <span
    className="mdl-layout-title"
  >
    My Portfolio
  </span>
  <nav
    className="mdl-navigation"
  >
    <a
      className="mdl-navigation__link"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className="mdl-navigation__link"
      href="/gallery"
      onClick={[Function]}
    >
      Gallery
    </a>
    <a
      className="mdl-navigation__link"
      href="/admin"
      onClick={[Function]}
    >
      Admin
    </a>
    <a
      className="mdl-navigation__link"
      href="/aboutme"
      onClick={[Function]}
    >
      About Me
    </a>
    <a
      className="mdl-navigation__link"
      href="/contact"
      onClick={[Function]}
    >
      Contact
    </a>
  </nav>
</div>
`;
